content/en/docs/languages/go/generated-code.md

```diff
@@ -114,71 +100,51 @@ In this context, `MsgA` is the single request from client to server, and `MsgB`
 
 These methods have the following signature on the generated client stub:
 
-`Foo(ctx context.Context, in *MsgA, opts ...grpc.CallOption) (<ServiceName>_FooClient, error)`
+`Foo(ctx context.Context, in *MsgA, opts ...grpc.CallOption) (grpc.ServerStreamClient[*MsgB], error)`
 
-In this context, `<ServiceName>_FooClient` represents the server-to-client `stream` of `MsgB` messages.
+In this context, `grpc.ServerStreamClient`  represents the client side of server-to-client stream of `MsgB` messages.
```

Comment:
looks like there is extra space after `grpc.ServerStreamClient`

Reply:
Done